File: URI Programs/C/1012.c
#include <stdio.h>

int main(){
	double a, b, c, triangle, circle, trapezium, square, rectangle;
	scanf("%lf%lf%lf", &a, &b, &c);
	triangle = a * c / 2;
	circle = 3.14159 * c * c;
	trapezium = (a + b) * c / 2;
	square = b * b;
	rectangle = a * b;
	printf("TRIANGULO: %.3lf\n", triangle);
	printf("CIRCULO: %.3lf\n", circle);
	printf("TRAPEZIO: %.3lf\n", trapezium);
	printf("QUADRADO: %.3lf\n", square);
	printf("RETANGULO: %.3lf\n", rectangle);
	return 0;
}

File: URI Programs/Python/1012.py
a, b, c = map(float, input().split())

tri = (a*c)/2
cir = 3.14159*c**2
tra = (a+b)*c/2
qua = b**2
ret = a*b

print("TRIANGULO: " + str("%.3f" %tri))
print("CIRCULO: " + str("%.3f" %cir))
print("TRAPEZIO: " + str("%.3f" %tra))
print("QUADRADO: " + str("%.3f" %qua))
print("RETANGULO: " + str("%.3f" %ret))